model/migrations/88cf753-conversation_state.sql

```diff
@@ -0,0 +1,14 @@
+--
+-- Table structure for table `conversation_state`
```

Comment:
Actually, previously we applied the `user_` prefix to tables that contain user data (as opposed to global thingpedia data). I think there might be some client-side genie-toolkit code that relies on that assumption when calling the dbproxy API.